src/app/book/add-book/add-book.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { Book } from '../book';
+import { BookService } from '../book.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-add-book',
+  templateUrl: './add-book.component.html',
+})
+export class AddBookComponent implements OnInit {
+
+  BookCategories = ['programming', 'biographies', 'Business', 'References'];
```

Comment:
Why is Book Categories are hardcoded? Categories should be fetched from DB using the API